selfdrive/car/subaru/interface.py

```diff
@@ -96,7 +96,8 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.steerActuatorDelay = 0.1
 
     elif candidate in (CAR.FORESTER_PREGLOBAL, CAR.OUTBACK_PREGLOBAL_2018):
-      ret.safetyConfigs[0].safetyParam = 1  # Outback 2018-2019 and Forester have reversed driver torque signal
+      # Outback 2018-2019 and Forester have reversed driver torque signal
+      ret.safetyConfigs[0].safetyParam |= Panda.FLAG_SUBARU_PREGLOBAL_REVERSED_DRIVER_TORQUE
```

Comment:
Are we sure about this?

```python
  CAR.FORESTER_PREGLOBAL: dbc_dict('subaru_forester_2017_generated', None),  # -1
  CAR.LEGACY_PREGLOBAL: dbc_dict('subaru_outback_2015_generated', None),  # 1 
  CAR.OUTBACK_PREGLOBAL: dbc_dict('subaru_outback_2015_generated', None),  # -1
  CAR.OUTBACK_PREGLOBAL_2018: dbc_dict('subaru_outback_2019_generated', None),  # -1
```

Reply:
We should update `test_models` to check driver torque consistency between openpilot and panda.